projects/igniteui-angular/src/lib/directives/toggle/README.md

```diff
@@ -53,7 +53,7 @@ handlers when the toggle is opened and respectively closed.
 
 # IgxToggleAction Directive
 
-The **IgxToggleAction** provides a way for user to Open/Close(toggle) every Component/Directive which implements **IToggleView** interface by providing the reference to this particular Component/Directive or ID which is registered into **IgxNavigationService**. It is also applicable upon **IgxToggle**
+The **IgxToggleAction** provides a way for user to Open/Close(toggle) every Component/Directive which implements **IToggleView** interface by providing the reference to this particular Component/Directive or ID which is registered into **IgxNavigationService**. It is also applicable upon **IgxToggle**. When applied to a Component/Directive **IgxToggleAction** forces **IToggleView**'s target to be the element where **IgxToggleAction** is set.
```

Comment:
It could be placed on plain HTML elements too, right? With much the same effect. So this should probably state that the toggle action will always set its host element as the position strategy target.

Reply:
done!